_nds/TWiLightMenu/unlaunch/backgrounds/_meta.json

```
@@ -63,6 +63,11 @@
 		"author": "SNBeast",
 		"categories": ["vista","windows"]
 	},
+	"codmw2": {
+		"title": "Call Of Duty : Modern Warfare 2",
+		"author": "Emran_Ahmed",
+		"categories": ["call of duty","games"]
```

Oh, can you make the category not have spaces? I think that'll mess with it

```suggestion
		"categories": ["call-of-duty","games"]
```
or
```suggestion
		"categories": ["cod","games"]
```

---

so i need to do another pr?

---

No, just make another commit to this branch. You can also just do the `Commit suggestion` on either of the suggestions I did in that comment.